e2e/support/helpers/e2e-embedding-sdk-assertion-helpers.ts

```diff
@@ -0,0 +1,42 @@
+import { popover, tableInteractive } from "./e2e-ui-elements-helpers";
+
+export function assertSdkNotebookEditorUsable(root: Cypress.Chainable = cy) {
+  cy.findByText("Orders").should("be.visible");
+
+  // Wait until the entity picker modal is visible
+  // Using `cy.contains` does not work in the iframe.
+  root.contains("Pick your starting data");
```

Comment:
I am almost sure this is a Cypress bug that has to do with iframes. Using `cy.contains` works fine in component tests as it's already in a `getSdkRoot().within()` block. However, in the e2e iframe test it cannot find `Pick your starting data` even though it is using `iframe.within()`, and you need to specify the iframe content directly 🤦🏻